lib/switchtec.c

```diff
@@ -552,4 +552,64 @@ float switchtec_die_temp(struct switchtec_dev *dev)
 	return temp / 100.;
 }
 
+
+int switchtec_bind_info(struct switchtec_dev *dev, struct switchtec_bind_status_out *bind_status, int phy_port )
+{
+	int ret;
+
+	struct switchtec_bind_status_in sub_cmd_id =
+	{ .sub_cmd = MRPC_PORT_INFO,
+	  .phys_port_id = phy_port
```

Comment:
That open brace { should be on the previous line. Please follow the style in the rest of the code.